.golangci.yml

```diff
@@ -0,0 +1,74 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  maligned:
+    suggest-new: true
+  depguard:
+    list-type: blacklist
+    packages:
+      # logging is allowed only by logutils.Log, logrus
+      # is allowed to use only in logutils package
+      - github.com/sirupsen/logrus
+  misspell:
+    locale: US
+    ignore-words:
+      - cancelled
+  lll:
+    # Default is 120. '\t' is counted as 1 character.
+    # set our project to 200, as things like v3_structs with inline comments end up being a touch long
+    # also, for anyone using vscode, use the following configs:
+    # "rewrap.wrappingColumn": 200 ... requires the rewrap plugin
+    # "editor.rulers": [200]
+    line-length: 200
+  gocognit:
+    # minimal code complexity to report, 30 by default
+    # we need to refact the cluster resource to change this number
+    min-complexity: 94
```

Comment:
it would be better to do a `nolint: rule` in the offending blocks thsi way we prevent new offending blocks of being added

Reply:
done